src/loaders/common/loader_helper.py

```diff
@@ -184,6 +184,69 @@ def is_upa_info_complete(upa_dir: str):
     return True
 
 
+def make_collection_source_dir(
+        root_dir: str,
+        collection_source_dir: str,
```

Comment:
do you think we can make `collection_source_dir` defaults to `loader_common_names.COLLECTION_SOURCE_DIR`? 
And we can probably remove `form_source_dir` and replace usage with this function.

Reply:
Updated!